packages/core/contracts/cross-chain-oracle/ChildMessengerInterface.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+interface ChildMessengerInterface {
```

Comment:
very good. I'll move this around in a follow on PR that implements this interface for the different networks.